src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Modal vue de base avec boutons google, facebook, apple non actifs
+ */
+type Props = {
+  setContenuModal: React.Dispatch<React.SetStateAction<"init" | "connexion" | "inscription" | 'password-forget'>>
+};
+
+const ModalAuthInitView = ({ setContenuModal}: Props ) => {
+
+  return (
+    <>
```

Comment:
Amélioration repérée avec Sonarlint : A fragment with only one child is redundant.sonarlint(typescript:S6749)

Reply:
corrigé